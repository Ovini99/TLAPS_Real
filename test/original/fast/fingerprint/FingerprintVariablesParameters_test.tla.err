@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:2
@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real/tlapm-main/lib/tlaps/bin

@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real/tlapm-main/lib/tlaps/bin

@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

File "./FingerprintVariablesParameters_test.tla", line 19, characters 1-7:
[ERROR]: Could not prove or check:
           ASSUME NEW VARIABLE x
           PROVE  \E y : y # y
File "./FingerprintVariablesParameters_test.tla", line 16, characters 1-7:
[ERROR]: Could not prove or check:
           ASSUME NEW VARIABLE x
           PROVE  \E y : y # x
File "./FingerprintVariablesParameters_test.tla", line 1, character 1 to line 21, character 36:
[ERROR]: 2/2 obligations failed.
There were backend errors processing module `"FingerprintVariablesParameters_test"`.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:2
@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:16:1:16:8
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real/tlapm-main/lib/tlaps/bin

@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # x

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:smt
@!!meth:time-limit: 5
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:zenon
@!!meth:time-limit: 10
@!!reason:false
@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:19:1:19:8
@!!status:failed
@!!prover:isabelle
@!!meth:auto; time-limit: 30
@!!reason:Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real/tlapm-main/lib/tlaps/bin

@!!already:true
@!!obl:ASSUME NEW VARIABLE x
PROVE  \E y : y # y

@!!END

File "./FingerprintVariablesParameters_test.tla", line 19, characters 1-7:
[ERROR]: Could not prove or check:
           ASSUME NEW VARIABLE x
           PROVE  \E y : y # y
File "./FingerprintVariablesParameters_test.tla", line 16, characters 1-7:
[ERROR]: Could not prove or check:
           ASSUME NEW VARIABLE x
           PROVE  \E y : y # x
File "./FingerprintVariablesParameters_test.tla", line 1, character 1 to line 21, character 36:
[ERROR]: 2/2 obligations failed.
There were backend errors processing module `"FingerprintVariablesParameters_test"`.
