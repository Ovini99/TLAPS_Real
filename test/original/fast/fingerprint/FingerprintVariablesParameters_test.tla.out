
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:25:59 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 FingerprintVariablesParameters_test.tla

(* loading fingerprints in ".tlacache/FingerprintVariablesParameters_test.tlaps/fingerprints" *)
(* created new ".tlacache/FingerprintVariablesParameters_test.tlaps/FingerprintVariablesParameters_test.thy" *)
(* fingerprints written in ".tlacache/FingerprintVariablesParameters_test.tlaps/fingerprints" *)

\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:26:00 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 FingerprintVariablesParameters_test.tla

(* loading fingerprints in ".tlacache/FingerprintVariablesParameters_test.tlaps/fingerprints" *)
(* created new ".tlacache/FingerprintVariablesParameters_test.tlaps/FingerprintVariablesParameters_test.thy" *)
(* fingerprints written in ".tlacache/FingerprintVariablesParameters_test.tlaps/fingerprints" *)
